pour/cup-finding.go

```diff
@@ -172,79 +181,83 @@ func calculateAverage(numbers []float64) float64 {
 	return sum / float64(len(numbers))
 }
 
-func (g *Gen) determineAdjustment(logger logging.Logger, inputX, inputY float64) (float64, float64) {
-	// deltaXNeg := 0.2
-	// deltaXPos := 0.2
-	// deltaYNeg := 0.295
-	// deltaYPos := 0.295
-
-	// [aw] consts before moving to config
-	// deltaXNeg := 0.2
-	// deltaXPos := 0.325
-	// deltaYNeg := 0.295
-	// deltaYPos := 0.325
-
-	deltaXNeg := g.conf.DeltaXNeg
-	deltaXPos := g.conf.DeltaXPos
-	deltaYNeg := g.conf.DeltaYNeg
-	deltaYPos := g.conf.DeltaYPos
-
-	logger.Infof("deltaXPos: %f", deltaXPos)
-	logger.Infof("deltaYPos: %f", deltaYPos)
-	logger.Infof("deltaXNeg: %f", deltaXNeg)
-	logger.Infof("deltaYNeg: %f", deltaYNeg)
-	logger.Infof("hi there lol")
-
-	// 313,225
-	deltaX := 313 - inputX
-	deltaY := 225 - inputY
-	if math.Abs(deltaX) < 7 {
-		deltaX = 0
-	}
-	logger.Infof("deltaX: %f", deltaX)
-	logger.Infof("deltaY: %f", deltaY)
-	if deltaX > 0 && deltaY > 0 {
-		logger.Info("deltaX > 0 && deltaY > 0")
-		logger.Info("using deltaXPos and deltaYPos")
-		return deltaX * deltaXPos, deltaY * deltaYPos
-	} else if deltaX > 0 && deltaY < 0 {
-		logger.Info("deltaX > 0 && deltaY < 0")
-		logger.Info("using deltaXPos and deltaYNeg")
-		deltaXPos = 0.22
-		return deltaX * deltaXPos, deltaY * deltaYNeg
-	} else if deltaX < 0 && deltaY < 0 {
-		logger.Info("deltaX < 0 && deltaY < 0")
-		logger.Info("using deltaXNeg and deltaYNeg")
-		return deltaX * deltaXNeg, deltaY * deltaYNeg
-	} else if deltaX == 0 && deltaY < 0 {
-		logger.Info("deltaX == 0 && deltaY < 0")
-		logger.Info("using 0 and deltaYNeg")
-		return deltaX * deltaXNeg, deltaY * deltaYNeg
-	} else if deltaY == 0 && deltaX < 0 {
-		logger.Info("deltaY == 0 && deltaX < 0")
-		logger.Info("using deltaXNeg and 0")
-		return deltaX * deltaXNeg, deltaY * deltaYNeg
-	} else if deltaX == 0 && deltaY > 0 {
-		logger.Info("deltaX == 0 && deltaY > 0")
-		logger.Info("using 0 and deltaYPos")
-		return 1, deltaY * deltaYPos
-	} else if deltaY == 0 && deltaX > 0 {
-		logger.Info("deltaY == 0 && deltaX > 0")
-		logger.Info("using deltaXPos and 0")
-		return deltaX * deltaXPos, deltaY * deltaYNeg
-	}
-	logger.Info("NONE OF THE CONDITINALS HIT, IN ELSE")
-	logger.Info("deltaX < 0 && deltaY > 0")
-	logger.Info("using deltaXNeg and deltaYPos")
-	return deltaX * 0, deltaY * deltaYPos
-}
+// func (g *Gen) determineAdjustment(logger logging.Logger, inputX, inputY, deltaXPos, deltaYPos, deltaXNeg, deltaYNeg float64) (float64, float64) {
```

Comment:
i'd just remove all code in this file that's commented out.
we don't need history.
git does that for us